minio/mc
# Added code to print the copy resume information

## Description
Before resuming the copy again, now it would print the below message

```
$ mc cp --continue --recursive ~/go/pkg/ myminio/source-bucket
 447.30 MiB / 1.82 GiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉━
Resuming copy from  447 MiB  /  1.8 GiB
...er_delegation_credential.go: 468.63 MiB / 1.82 GiB ━━━━━━━━━━━━━━━━━━━━━━━━━━━━┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉━ 4.70 MiB/s
```

## Motivation and Context
better reporting of resuming the copy
github issue: https://github.com/minio/mc/issues/4213

## How to test this PR?
Follow the below step to test the scenario
Step-1: Create a bucket using minio console
Step-2: Start copying a huge set of object from say local dir to the bucket using command `mc cp --recursive <source dir> <minio cluster name>/<bucket name>`. Stop the copying midway pressing Ctrl+C
Step-3: Resume the copy using the command `mc cp --continue --recursive <source dir> <minio cluster name>/<bucket name>`. It should show the additional info about resuming the copy as below after calculating the already copied data
```
Resuming copy from  <Size already copied>  / <Total size to be copied>
```

## Types of changes

## Checklist:
